### ISSUE TITLE
Syntax highlighting applied to wrong kind of output.

### ISSUE THREAD

**Issue by yminsky** [realworldocaml/book#2832]
In this example from the guided tour, syntax highlighting is applied to the wrong scope:

![image](https://user-images.githubusercontent.com/715302/35309624-6a115a42-007b-11e8-96d3-9816fb6c8e98.png)

Note that the underlying expect declaration separates out the output that should be highlighted, and the part that shouldn't:

``` ocaml
[@@@part "12"];;
let big_number x = x > 3;;
[%%expect ocaml {|val big_number : int -> bool = <fun>|}];;
first_if_true big_number 4 3;;
[%%expect ocaml {|- : int = 4|}];;
[@@@part "13"];;
first_if_true big_number "short" "loooooong";;
[%%expect{|
Characters 25-32:
Error: This expression has type string but an expression was expected of type
         int
|}];;
```

Only the `[%%expect ocaml]` sections should be highlighted, not the plain `%%expect` declarations

**Comment by avsm** [comment]
Yes, this needs to be fixed in the rwo tooling indeed.

**Comment by samoht** [comment]
The right fix for this is to write a [prism.js plugin](http://prismjs.com/#plugins). A good starting base would be the [command-line](http://prismjs.com/plugins/command-line/) which might be adaptable to expect tests without too much trouble.

**Comment by samoht** [comment]
I've added a quick fix for OCaml snippets in #2937 but this is not perfect as it adds extra `#` on multi-lines inputs.
 
But it would probably be simpler to do the rendering on the server-side...

**Comment by NathanReb** [comment]
closing as it has been fixed!
